From 5227e2d0de2fb7a0697db817b589ab671b68b834 Mon Sep 17 00:00:00 2001
From: Etan Cohen <etancohen@google.com>
Date: Thu, 14 Sep 2017 08:42:12 -0700
Subject: [PATCH 03/16] msm8974-common: Uprev Wi-Fi HAL to 1.2

New HAL APIs are planned. This requires a HAL uprev.

Bug: 65647646
Test: builds and Wi-Fi runs
Change-Id: Ib7efd3d5263e73d1ad40e1a5a687bdb6c133c812
---
 manifest.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/manifest.xml b/manifest.xml
index 65f41a8..c05ac8f 100644
--- a/manifest.xml
+++ b/manifest.xml
@@ -152,7 +152,7 @@
     <hal format="hidl">
         <name>android.hardware.wifi</name>
         <transport>hwbinder</transport>
-        <version>1.0</version>
+        <version>1.2</version>
         <interface>
             <name>IWifi</name>
             <instance>default</instance>
-- 
2.17.1

